tests/player_helpers.cpp

```diff
@@ -72,7 +72,9 @@ void clear_character( player &dummy )
     // This sets HP to max, clears addictions and morale,
     // and sets hunger, thirst, fatigue and such to zero
     dummy.environmental_revert_effect();
-    dummy.set_stored_kcal( dummy.get_healthy_kcal() ); // But not stored kcal
+    // However, the above does not set stored kcal;
+    // 2170 is calories of debug_nutrition
```

Comment:
Stick a `REQUIRE` or something here guaranteeing that?

Reply:
Point - but it's erroring in stomach contents bmr requirements, so I may need to revert that anyway.